pkgs/by-name/cl/clever-tools/package.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, nodejs_18
+}:
+
+buildNpmPackage rec {
+  pname = "clever-tools";
+
+  version = "3.7.0";
+
+  nodejs = nodejs_18;
+
+  src = fetchFromGitHub {
+    owner = "CleverCloud";
+    repo = "clever-tools";
+    rev = "${version}";
```

Contributing guide advises against this
```suggestion
    rev = version;
```

---

Bad habits die hard, this should be fixed now, thanks!